### PR TITLE
Submit PR on spec

### PR DESCRIPTION
**Pull request by jkrech**
Rules to submit PR on the spec https://github.com/Open-CMSIS-Pack/Open-CMSIS-Pack-Spec/issues/132

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,3 +62,28 @@ In the spirit of openness we will be tagging issues with the following:
...
+
+### Pull Requests
+When raising a Pull Request (PR) changing the PACK.xsd schema file, please ensure that the version, date of change, release-notes and 
+documentation get updated consistently throughout your PR. 
+
+Rules: 
+- Breaking/incompatible changes are not accepted and require further discussion
+
+Exceptions: 
+- Extending the `DeviceVendorEnum` does not require a version increment of the schema version.
```
Comment (JonatanAntoni) [inline README.md:74]: This is only half the truth. @KeilChris realized that new vendors might require a new schema version tag to be able to refer to the updated schema.

**Review by JonatanAntoni — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (KeilChris) [reply README.md:74]: If they use the genpack library, they can point the gen_pack.sh script to read a certain tagged version of the PACK.xsd. If you do not update the number and create a new tagged version, you would need to overwrite the tagged version. Is that possible?

Comment (JonatanAntoni) [reply README.md:74]: Yes, an admin can overwrite existing tags but this is often considered bad practice.

Comment (KeilChris) [reply README.md:74]: I thought so! That's the reason I tend to update the version number.

Comment (jkrech) [reply README.md]: I removed the exceptions section.

Comment (KeilChris) [reply README.md]: Thanks! (And I'm sorry)

**Review by KeilChris — APPROVED** [review]
LGTM

**Review by JonatanAntoni — APPROVED** [review]

**Pull request merged**